src/graphics/color.js

```diff
@@ -135,16 +135,25 @@ Crafty.extend({
 
 // Define some variables required for webgl
 var fs = require('fs');
-var COLOR_VERTEX_SHADER = fs.readFileSync(__dirname + '/shaders/color.vert', 'utf8');
-var COLOR_FRAGMENT_SHADER = fs.readFileSync(__dirname + '/shaders/color.frag', 'utf8');
-var COLOR_ATTRIBUTE_LIST = [
-    {name:"aPosition", width: 2},
-    {name:"aOrientation", width: 3},
-    {name:"aLayer", width:2},
-    {name:"aColor",  width: 4}
-];
-
 
+Crafty.defaultShader("Color", new Crafty.WebGLShader(
+    fs.readFileSync(__dirname + '/shaders/color.vert', 'utf8'),
+    fs.readFileSync(__dirname + '/shaders/color.frag', 'utf8'),
+    [
+        { name: "aPosition",    width: 2 },
+        { name: "aOrientation", width: 3 },
+        { name: "aLayer",       width: 2 },
+        { name: "aColor",       width: 4 }
+    ],
+    function(e, _entity) {
+        e.program.writeVector("aColor",
+            entity._red/255,
```

Comment:
Maybe you mean `entity`, not `_entity` here?

(Unfortunately we don't have unit tests for WebGL right now.)